app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -93,6 +95,16 @@ class ConversationViewModel @Inject constructor(
         }
     }
 
+    fun navigateToInitiatingCallScreen() {
```

Comment:
here it mentions to navigate to InitiatingCallScreen but in the end we are navigating to CallEstablished

Reply:
For now, we will navigate the user directly to CallEstablishedScreen..In next iteration, we will update it to InitiatingCallScreen

Reply:
could you add what you wrote above as a comment ? explains it perfectly